xtra/tests/basic.rs

```diff
@@ -6,6 +6,7 @@ use std::time::Duration;
 use futures_util::task::noop_waker_ref;
 use futures_util::FutureExt;
 use smol_timeout::TimeoutExt;
+use tokio::task::JoinSet;
```

Comment:
side note: it's funny how many async crates we're mixing and matching here, haha